src/Action/SyliusConvertAction.php

```diff
@@ -42,6 +53,11 @@ public function execute($request)
 
         $model = ArrayObject::ensureArrayObject($payment->getDetails());
 
+        //If there is a new token, clear all details
+        if (isset($model['success_url']) && basename($model['success_url']) !== $request->getToken()->getHash()) {
+            $model = new ArrayObject();
+        }
```

Comment:
I double check this code and I don't think this is a good idea resetting the `$details` here because the sucess_url and failure_url are set previously by https://github.com/ekyna/PayumMonetico/blob/master/src/Action/CaptureAction.php#L37.

Then If you could delete this I could merge this PR.